content/en/infrastructure/livecontainers.md

```diff
@@ -21,53 +21,58 @@ Taking inspiration from bedrock tools like *htop*, *ctop*, and *kubectl*, live c
 
 Coupled with integrations for [Docker][2], [Kubernetes][3], [ECS][4], and other container technologies, plus built-in tagging of dynamic components, the live container view provides a detailed overview of your containers' health, resource consumption, logs, and deployment in real time:
 
-{{< img src="infrastructure/livecontainers/livecontainerssummaries.png" alt="Live containers with summaries"  >}}
+{{< img src="infrastructure/livecontainers/livecontainersoverview.png" alt="Live containers with summaries"  >}}
 
-## Installation
+### Kubernetes Resources
 
-After deploying the [Docker Agent][5], container metrics are available without additional configuration. To enable log collection follow these steps:
+[Kubernetes Resources for Live Containers][1] is currently in private beta. Fill out [this form][5] to request access.
 
-{{< tabs >}}
+If you're using Kubernetes, enable Kubernetes Resources for Live Containers to gain multi-dimensional visibility into all Kubernetes workloads across your clusters. Inspired by the `kubectl` tool, this feature gives you complete coverage of your Kubernetes infrastructure in a continuously updated table with curated resource metrics, faceted search, pre-workload detailed view, and visualized maps.
 
-{{% tab "Linux/Windows" %}}
-Once the [Datadog Agent][1] is installed, enable log collection by editing the [Agent main configuration file][2] and updating the following parameters:
+## Installation
 
-```yaml
-logs_enabled: true
-listeners:
-  - name: docker
-config_providers:
-  - name: docker
-    polling: true
-```
+Follow the [Docker][6] or [Kubernetes][7] Agent installation instructions. Container metrics are available without additional configuration after installation.
 
-**Notes**:
+**Kubernetes Resources for Live Containers requires installation of**:
 
-* To collect container information in the standard install rather than with the [Docker Agent][1], the `dd-agent` user must have permissions to access **docker.sock**.
-* Logs are indexed by default, however [Exclusion Filters][2] are configurable for fine-grained controls over indexing and uniquely receiving Live Tail data.
+* [Datadog Agent][8] version 7.21.1 (or above)
+* [Datadog Cluster Agent][9] 1.8.0 (or above)
 
-[1]: /agent/docker/log/?tab=hostinstallation
-[2]: /agent/guide/agent-configuration-files/
-{{% /tab %}}
+### Kubernetes Resources
 
-{{% tab "Docker" %}}
+To enable Kubernetes Resources for Live Containers, follow the [Helm instructions][10] and add the following changes to your `values.yaml` file:
 
-Follow the instructions for the [Docker Agent][1], passing in the following attributes, in addition to any other custom settings as appropriate:
+{{< code-block lang="yaml" >}}
+datadog:
+   ...
+   orchestratorExplorer:
+      enabled: true
+   ...
+   agents:
+      image:
+        repository: datadog/agent
+        tag: 7.21.1
+        pullPolicy: Always
+   ...
+   clusterAgent:
+	repository: datadog/cluster-agent
+tag: 1.7.0
```

Comment:
just to doublecheck. If this corresponds to the DCA version for orchestrator explorer, it should be `1.8.0`

Reply:
actually both tags should be `tag: latest` now, and we specify the correct version in the requirements above (1.8.0 indeed)

Reply:
we also have an example for the correct values in https://docs.google.com/document/d/1LiIvmynJ4hPsVytWSZXM3X3PZfXhVW9j0zcvVb3pi60/edit#

Reply:
also note that lines 58-60 are not indented correctly